requirements/dev.in

```diff
@@ -0,0 +1,9 @@
+# Additional requirements for development of this application
+-c constraints.txt
+
+-r base.in                # Dependencies for edx-completion itself
+-r quality.in             # Dependencies for code quality checks
+-r test.in                # Dependencies for test case execution
+
+pip-tools                 # Requirements file management
```

Comment:
This should be in a file called pip-tools.in/pip-tools.txt

Reply:
pip-tools.in file created. pip-tools.txt will come from pip-compile.